src/index.tsx

```diff
@@ -1,11 +1,11 @@
-import React from 'react';
```

Comment:
use single quotes instead of double quotes

Reply:
fixed